.github/workflows/coordinator.yml

```diff
@@ -38,6 +40,12 @@ jobs:
   build-and-test:
     runs-on: ubuntu-22.04
 
+    services:
+      redis:
+        image: redis:7
+        ports:
+          - 6379:6379
```

Comment:
1. What does `redis:7` mean? Why the 7?
2. Why choose `6379` out of interest?

Reply:
1. it is the version. I increased it to redis 8 which is the current one. Thanks for the catch 🚀 
2. 6379 is the default redis port